src/Core/Grid/Query/CustomerGroupsQueryBuilder.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace PrestaShop\PrestaShop\Core\Grid\Query;
+
```

Comment:
You must use php strict typing for new classes

Reply:
license header